### ISSUE TITLE
emitter module wrong reference?

### ISSUE THREAD

**Issue by KapilGahlot** [juliangruber/stream#5]
Hi,

I see below require statement in your index.js and you have listed emitter-component as one of your dependencies. Should the below line require from emitter-component instead of emitter?

var Emitter = require('emitter');

Your module is a dependency of another module that one of my Meteor apps require. My app threw an error that Emitter is not an constructor but when I changed the require reference as mentioned above, the error got resolved. I am curious that I did not see a similar issue reported by anybody else so not sure how it is working for others?

Regards

**Comment by juliangruber** [comment]
Interesting. Hm, this module hasn't been updated in a while and I haven't used component in a while. Happy to accept any PR that fixes this

**Comment by durdenx** [comment]
I fixed the issue in #6

**Comment by barbalex** [comment]
I am getting this error when building using vite:
```
"emitter" is imported by "emitter?commonjs-external", but could not be resolved – treating it as an external dependency
```
Which then breaks preview and production 😢

**Comment by barbalex** [comment]
Please merge https://github.com/juliangruber/stream/pull/6, this is blocking our project.

**Comment by barbalex** [comment]
For any one else running into this: use path-package (https://www.npmjs.com/package/patch-package) to fix the import as done in https://github.com/juliangruber/stream/pull/6/files

**Comment by barbalex** [comment]
@lucasriondel no, can't remember having that issue. The project that had our issue is being rebuilt and this part has not been added yet. So I'm not quite up to date right now.

**Comment by lucasriondel** [comment]
@barbalex ok so just so you know, using this package on modern version of node is clearly not recommended. I was able to remove it completely and using the built-in `stream` package from `node` instead. Everything worked out of the box.

**Comment by barbalex** [comment]
@lucasriondel Our app is a web app. There is no node to use it's built-in stream. For a node app your solution makes a lot of sense.
